src/CategoryCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Card, CardHeader, IconButton, Avatar } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Just wondering: but maybe we want to stick to a single way of customizing components? We need Material UI for components, but for adding our own styles or overriding Material styles we could choose Styled Components. That said,  `useStyles` will work perfectly fine here - I'm suggesting a single styling tool just for the sake of consistency in the app.

Reply:
I've introduced Styled Components to the CategoryCard. It's my first contact with it, so please check out if I applied it correctly.

Reply:
This is exactly how we should use `styled-components`! :) Working fine!